server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -2298,7 +2299,7 @@ public boolean isMajorCompactionQueued() {
       KeyExtent high = new KeyExtent(extent.getTableId(), extent.getEndRow(), midRow);
 
       String lowDirectory = createTabletDirectory(context, getTabletServer().getFileSystem(),
-          extent.getTableId(), midRow);
```

Comment:
Why would this change?

Reply:
This was changed to allow passing the extent down to use the new constructor. Because of earlier requested changes the value passed to the VolumeChooserEnvironment is a TabletId, which takes an extent

Reply:
Understood now. See below comment.